src/conventions/h-group/interpret-discard.js

```diff
@@ -97,15 +98,25 @@ export function interpret_discard(game, action, card) {
 	// Discarding a useful card
 	// Note: we aren't including chop moved and finessed cards here since those can be asymmetric.
 	// Discarding with a finesse will trigger the waiting connection to resolve.
-	if (card.clued && rank > state.play_stacks[suitIndex] && rank <= state.max_ranks[suitIndex]) {
-		logger.warn('discarded useful card!');
-		common.restore_elim(card);
-
-		// Card was bombed
-		if (failed)
-			undo_hypo_stacks(game, identity);
-		else
-			interpret_sarcastic(game, action);
+	if (rank > state.play_stacks[suitIndex] && rank <= state.max_ranks[suitIndex]) {
+		if (card.clued) {
+			logger.warn('discarded useful clued card!');
+			common.restore_elim(card);
+
+			// Card was bombed
+			if (failed)
+				undo_hypo_stacks(game, identity);
+			else
+				interpret_sarcastic(game, action);
+		} else if (game.level >= LEVEL.STALLING) {
```

Comment:
We can be in dda after a discard/bomb too. We can only rule this out if we see the other copy or we determine that it was a *Sarcastic Discard* to us.